Firefox: about:crashes don't work

**Describe the bug**
Firefox crashed because my disk was full. I wanted to submit a crash report, but the site the documentation mentions don't exist.

https://support.mozilla.org/en-US/kb/mozillacrashreporter

**To Reproduce**
Steps to reproduce the behavior:
1. open `about:crashes` in firefox

**Expected behavior**
firefox works like an official release

**Screenshots**
![Screenshot from 2020-12-29 17-21-55](https://user-images.githubusercontent.com/91113/103299630-41894200-49fd-11eb-9cba-28f04c80ef9c.png)

**Additional context**
Add any other context about the problem here.

**Notify maintainers**
@eelco

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.1, NixOS, 21.03.git.173696f216f (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.10`
 - channels(root): `"home-manager-20.09, nixos-20.09.2344.a3a3dda3bac, nixos-hardware, nixos-unstable-21.03pre257780.e9158eca70a"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

The build starts with that, but fails exactly at the crashreporter. I guess no one has tested that in a while.

```
   Compiling dump_syms_rust_demangle v0.1.0 (/build/firefox-84.0.2/toolkit/crashreporter/rust)
     Running `CARGO=/nix/store/fd62fkbclh3s0jym0q6ia3fgd8mzhxzi-cargo-1.44.1/bin/.cargo-wrapped CARGO_MANIFEST_DIR=/build/firefox-84.0.2/toolkit/crashreporter/rust CARGO_PKG_AUTHORS= CARGO_PKG_DESCRIPTION= CARGO_PKG_HOMEPAGE= CARGO_PKG_NAME=dump_syms_rust_demangle CARGO_PKG_REPOSITORY= CARGO_PKG_VERSION=0.1.0 CARGO_PKG_VERSION_MAJOR=0 CARGO_PKG_VERSION_MINOR=1 CARGO_PKG_VERSION_PATCH=0 CARGO_PKG_VERSION_PRE= LD_LIBRARY_PATH='/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/release/deps:/nix/store/wzk6j6ijc0y91q30kndr8m9frlf90fsx-rustc-1.44.1/lib' /nix/store/wzk6j6ijc0y91q30kndr8m9frlf90fsx-rustc-1.44.1/bin/rustc --crate-name dump_syms_rust_demangle toolkit/crashreporter/rust/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type staticlib --emit=dep-info,link -C opt-level=2 -C panic=abort -C metadata=7cbd7ef9159eddf3 -C extra-filename=-7cbd7ef9159eddf3 --out-dir /build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/x86_64-unknown-linux-gnu/release/deps --target x86_64-unknown-linux-gnu -C linker=/build/firefox-84.0.2/build/cargo-linker -L dependency=/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/x86_64-unknown-linux-gnu/release/deps -L dependency=/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/release/deps --extern rustc_demangle=/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/x86_64-unknown-linux-gnu/release/deps/librustc_demangle-5f31f71fe6b2c40e.rlib -C opt-level=2 -C debuginfo=2 --cap-lints warn -C codegen-units=1`
    Finished release [optimized] target(s) in 1.33s
make[3]: Leaving directory '/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/toolkit/crashreporter/rust'
make[3]: Entering directory '/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/toolkit/crashreporter/google-breakpad/src/tools/linux/dump_syms'
toolkit/crashreporter/google-breakpad/src/tools/linux/dump_syms/dump_syms
/nix/store/azayfhqyg9mjyv6rv9bcypi6ws8aqfmy-gcc-wrapper-9.3.0/bin/g++ -std=gnu++17 -o ../../../../../../../dist/host/bin/dump_syms -freorder-blocks -O2 -O2 -g  @/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/toolkit/crashreporter/google-breakpad/src/tools/linux/dump_syms/dump_syms.list /build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/x86_64-unknown-linux-gnu/release/libdump_syms_rust_demangle.a -ldl -lpthread -lrt
/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/_virtualenvs/init_py3/bin/python -m mozbuild.action.check_binary --host ../../../../../../../dist/host/bin/dump_syms
BUILDTASK {"argv": ["/build/firefox-84.0.2/python/mozbuild/mozbuild/action/check_binary.py", "--host", "../../../../../../../dist/host/bin/dump_syms"], "start": 1610741244.609147, "end": 1610741244.6099205, "context": null}
make[3]: Leaving directory '/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/toolkit/crashreporter/google-breakpad/src/tools/linux/dump_syms'
collect2: error: ld returned 1 exit status
make[3]: *** [/build/firefox-84.0.2/config/rules.mk:545: libxul.so] Error 1
make[3]: Leaving directory '/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu/toolkit/library/build'
make[2]: *** [/build/firefox-84.0.2/config/recurse.mk:72: toolkit/library/build/target] Error 2
make[2]: Leaving directory '/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu'
make[1]: *** [/build/firefox-84.0.2/config/recurse.mk:34: compile] Error 2
make[1]: Leaving directory '/build/firefox-84.0.2/obj-x86_64-pc-linux-gnu'
make: *** [/build/firefox-84.0.2/config/rules.mk:355: default] Error 2
builder for '/nix/store/27cpp1va9ia77kc6ragbg195m880s1fx-firefox-unwrapped-84.0.2.drv' failed with exit code 2
cannot build derivation '/nix/store/wpsyb3imafddymm73023k6783gb2y3si-firefox-84.0.2.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/lbgnr6ygqpzh6y7vnal75nwa3lih1gyk-system-path.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/0xk6ds5w65i9hjky2b2z15g9fxw8b6qj-nixos-system-gaming-20.09.2618.352481146d8.drv': 1 dependencies couldn't be built
error: build of '/nix/store/0xk6ds5w65i9hjky2b2z15g9fxw8b6qj-nixos-system-gaming-20.09.2618.352481146d8.drv' failed
```
It took 33 minutes to get to that point with a i9 cpu (16x 4GHz). 32 GB RAM would be good, 16 will SWAP.

--- a/pkgs/applications/networking/browsers/firefox/common.nix
+++ b/pkgs/applications/networking/browsers/firefox/common.nix
@@ -94,6 +94,7 @@
 # WARNING: NEVER set any of the options below to `true` by default.
 # Set to `!privacySupport` or `false`.
 
+, crashreporterSupport ? !privacySupport
 , geolocationSupport ? !privacySupport
 , googleAPISupport ? geolocationSupport
 , webrtcSupport ? !privacySupport
@@ -106,10 +107,6 @@
 # Controlling the nagbar and widevine CDM at runtime is possible by setting
 # `browser.eme.ui.enabled` and `media.gmp-widevinecdm.enabled` accordingly
 , drmSupport ? true
-
-## other
-
-, crashreporterSupport ? false
 
 # As stated by Sylvestre Ledru (@sylvestre) on Nov 22, 2017 at
 # https://github.com/NixOS/nixpkgs/issues/31843#issuecomment-346372756 we

